tests/apollo/test_skvbc_st_reply_size.py

```diff
@@ -0,0 +1,107 @@
+# Concord
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+import os.path
+import unittest
+from os import environ
+
+import trio
+
+from util import skvbc as kvbc
+from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX
+
+
+def build_cmd(reply_size=None, page_size=None, num_pages=None):
+    def start_replica_cmd(builddir, replica_id):
+        """
+        Return a command that starts an skvbc replica when passed to
+        subprocess.Popen.
+        """
+        statusTimerMilli = "500"
+        path = os.path.join(builddir, "tests", "simpleKVBC", "TesterReplica", "skvbc_replica")
+        cmd = [
+          path,
+          "-k", KEY_FILE_PREFIX,
+          "-i", str(replica_id),
+          "-s", statusTimerMilli,
+        ]
+        if os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower() in set(["true", "on"]):
+          cmd.append("-p")
+
+        if reply_size is not None:
+          cmd.extend(["--max-reply-message-size", str(reply_size)])
+        if page_size is not None:
+          cmd.extend(["--max-num-of-reserved-pages", str(num_pages)])
+        if page_size is not None:
+          cmd.extend(["--size-of-reserved-page", str(page_size)])
+
+        return cmd
+    return start_replica_cmd
+
+class SkvbcStReplySize(unittest.TestCase):
+
+    __test__ = False  # so that PyTest ignores this test scenario
+
+    async def state_transfer_after_first_checkpoint(self, bft_network):
```

Comment:
Let's make this a private method prefixed with `_`, and move it after the actual test methods that invoke it.